tenant_schemas/tests/test_tenants.py

```diff
@@ -1,60 +1,109 @@
-import json
-from StringIO import StringIO
-
 from django.conf import settings
 from django.contrib.auth.models import User
-from django.core.management import call_command
-from django.db import connection
+from django.db import connection, transaction
```

Comment:
Needed to verify that tenant is created atomically in test test_tenant_schema_is_created_atomically.